src/reducers/comments.tsx

```diff
@@ -141,6 +200,37 @@ const reducer: Reducer<CommentsState, ActionType<typeof actions>> = (
           [key]: {
             ...info,
             beginNewComment: true,
+            savingComment: false,
+          },
+        },
+      };
+    }
+    case getType(actions.beginSaveComment): {
+      const key = createCommentKey(action.payload);
```

Comment:
These two lines are repeated throughout this file. Might it be worthwhile to create a helper which executes these two lines are returns `{key, info}`?

Reply:
Sure